workshops/erc20-mts/scripts/cli-keys.sh

```
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+source .local.env
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)